app/controllers/repos_controller.rb

```
@@ -18,4 +20,12 @@ def index
       end
     end
   end
+
+  private
+
+  def save_token
+    if current_user.token.blank?
+      current_user.update(token: session[:github_token])
```

Should this line be `update!` to not swallow any potential errors?